src/grt/src/GlobalRouter.cpp

```diff
@@ -1071,8 +1071,11 @@ void GlobalRouter::computeTrackAdjustments(int min_routing_layer,
 
 void GlobalRouter::computePinOffsetAdjustments()
 {
-  for (auto const& map_obj : pad_pins_connections_) {
-    for (const auto& segment : map_obj.second) {
+  for (auto& net_route_fake_pins : pad_pins_connections_) {
```

Comment:
You can use `for (auto& [db_net, segments] : pad_pins_connections_` to avoid using the `.first` and `.second` below.